ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_HasType_User()
+        {
+            //Arrange
+            User _user = new User("Tommy");
+            string title = null;
+            //Act
+            Article _article = new Article(_user);
+            //Assert
+            Assert.Null(title);
+            Assert.IsType<User>(_article.Autor);
+        }
+        [Fact]
+        public void CtorArticle_HasTitleAndText_TypeString()
+        {
+            //Arrange
+            string title = "Title";   
+            User _user = new User();
+            //Act
+            Article _article = new Article(title);
+            //Assert
+            Assert.IsType<string>(_article.Title);
+            Assert.NotNull(_user);
```

Comment:
Here is similar to previous comment.
1. The rows #28 and #33 should be deleted as they do nothing.
2. Do not use checking the type (`IsType<>()` method) when the `//Act` part does not return the result with different types. The Title property has a predefined type and always return string. You could test that this value is not null instead but it is already done within the `Ctor_AutorTitleText_NotNull()` test.

So just delete this test or use other test like:
```csharp
  //Arrange
  string title = "Title";
  User user = new User();
  //Act
  Article _article = new Article(title, user);
  //Assert
  Assert.NotNull(_article.Title);
  Assert.NotNull(_article.Autor);
  Assert.Equal(_article.Title, title);
  Assert.Equal(_article.Autor, user);
```